networking/net_plugin.go

```diff
@@ -116,6 +119,7 @@ func (e *podEnv) execNetPlugin(cmd string, n *activeNet, netns string) ([]byte,
 	}
 
 	vars := [][2]string{
+		{"CNI_VERSION", "0.3.0"},
```

Comment:
AFAIR before `0.3.0` there was no version checking, so introducing this there should not broke vendored CNI version, but tbh - i'm not sure. @steveeJ am i correct?